### PR TITLE
doc: add references to java 21 image

### PR DESCRIPTION
**Pull request by sparsick**
I saw that you already support Java 21 but this is not mention in the documentation.

### CODE REVIEW

**Comment by loosebazooka** [comment]
We're not quite at the point to finalize java21. There are a few minor details to work out

**Comment by mkarg** [comment]
Any news on the Java 21 topic? :-)

**Review by andrzejsydor — APPROVED** [review]

**Comment by mkarg** [comment]
@sparsick So finally your PR got approved, but meanwhile it is conflicting. Kindly asking you to resolve, so it can finally get merged with the main branch. :-)

**Comment by sparsick** [comment]
@mkarg  Thanks for mentioning. It is resolved.

File: README.md [hunk 1]
```diff
@@ -39,6 +38,7 @@ The following images are currently published and updated by the distroless proje
 | gcr.io/distroless/python3-debian12    | latest, nonroot, debug, debug-nonroot | amd64, arm64                      |
 | gcr.io/distroless/java-base-debian12  | latest, nonroot, debug, debug-nonroot | amd64, arm64, s390x, ppc64le      |
 | gcr.io/distroless/java17-debian12     | latest, nonroot, debug, debug-nonroot | amd64, arm64, s390x, ppc64le      |
+| gcr.io/distroless/java21-debian12     | latest, nonroot, debug, debug-nonroot  | amd64, arm64, ppc64le            |
```
Comment (mkarg) [inline README.md:41]: @sparsick Just noticed... maybe you like to remove the excess blank. 😄

**Review by mkarg — APPROVED** [review]
Thanks for contributing! ❤️

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ The following images are currently published and updated by the distroless proje
 | gcr.io/distroless/python3-debian12    | latest, nonroot, debug, debug-nonroot | amd64, arm64                      |
 | gcr.io/distroless/java-base-debian12  | latest, nonroot, debug, debug-nonroot | amd64, arm64, s390x, ppc64le      |
 | gcr.io/distroless/java17-debian12     | latest, nonroot, debug, debug-nonroot | amd64, arm64, s390x, ppc64le      |
-| gcr.io/distroless/java21-debian12     | latest, nonroot, debug, debug-nonroot  | amd64, arm64, ppc64le            |
+| gcr.io/distroless/java21-debian12     | latest, nonroot, debug, debug-nonroot | amd64, arm64, ppc64le             |
```
Comment (mkarg) [inline README.md:41]: LOL ;-)

**Comment by loosebazooka** [comment]
FYI @andrzejsydor is not a project maintainer. Not really sure why they keep approving PRs here.

**Comment by mkarg** [comment]
> FYI @andrzejsydor is not a project maintainer. Not really sure why they keep approving PRs here.

Understood.

So kindly requesting *your* review / merge instead! :-)

**Comment by loosebazooka** [comment]
We're still not fully stabilized on the java21 image. I understand that we're going slow here, but you're welcome to use it before then.

**Comment by ranma2913** [comment]
What's left to do on this? I can help.

File: README.md (hunk 1 above)
Comment (ranma2913) [reply README.md]: Is this causing a failure on the lint check? I don't see those actions running.

File: java/README.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ Specifically, the image contains everything in the [base image](../base/README.m
 
 * OpenJDK 11 (`gcr.io/distroless/java11-debian11`) and its dependencies.
 * OpenJDK 17 (`gcr.io/distroless/java17-debian12`, `gcr.io/distroless/java17-debian11`) and its dependencies.
+* OpenJDK 21 (`gcr.io/distroless/java21-debian12`) and its dependencies.
```
Comment (loosebazooka) [inline java/README.md:11]:
```diff
+ * Temurin OpenJDK 21 (`gcr.io/distroless/java21-debian12`) and its dependencies.
- * OpenJDK 21 (`gcr.io/distroless/java21-debian12`) and its dependencies.
```

**Comment by loosebazooka** [comment]
I think we're ready to merge this, just want to make it explicit that we're using temurin

**Pull request closed without merging**